eungabi/src/commonMain/kotlin/com/easternkite/eungabi/navigation/EunGabiEntry.kt

```diff
@@ -19,24 +19,82 @@ import com.easternkite.eungabi.utils.randomUuid
 import com.easternkite.eungabi.utils.withScheme
 import com.eygraber.uri.Uri
 
+/**
+ * The EunGabi navigation entry.
+ * It holds information of a backStack entry states to retrieve in [EunGabiNavHost].
+ */
 data class EunGabiEntry(
+    /**
+     * The destination of the entry.
+     */
     val eunGabiDestination: EunGabiDestination,
+
+    /**
+     * The unique identifier of the entry.
+     */
     val id: String = randomUuid,
+
```

Comment:
🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list
<!-- __reviewdog__:ChA2YWQ1N2YzZWM5YWU0YzA0EgZrdGxpbnQ= -->